features/dd-sdk-android-session-replay/src/main/kotlin/androidx/appcompat/widget/ActionBarContainerDrawableAccessor.kt

```diff
@@ -0,0 +1,26 @@
+@file:Suppress("PackageNaming")
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package androidx.appcompat.widget
+
+import android.annotation.SuppressLint
+import android.graphics.drawable.Drawable
+
+internal class ActionBarContainerDrawableAccessor(
```

Comment:
minor: maybe add `Datadog` to the name? unlikely that the class named `ActionBarContainerDrawableAccessor` will ever appear in Android codebase, but who knows.